eva-peng/week2/wed/homework-catwalk/js/main.js

```diff
@@ -0,0 +1,26 @@
+//Cat Walk Homework
+
+
+// Change the style of the img to have a "left" of "0px", so that it starts at the left hand of the screens. - see css file
+
+// Create a function called catWalk() that moves the cat 10 pixels to the right of where it started, by changing the "left" style property.
+
+
+// Call that function every 50 milliseconds. Your cat should now be moving across the screen from left to right. Hurrah!
+
+
+const catWalk = function () {
+    const catPic = document.getElementsByTagName('img')[0];// create a variable to store the img tag of the cat
+    let currentLeft = parseInt(catPic.style.left);
+    catPic.style.left = (currentLeft + 10) + 'px';
+    if (currentLeft > (window.innerWidth - catPic.width)) {
+    catPic.style.left = '0px';
+        
```

Comment:
if you want to turn the cat around i would suggest putting in the below
`catPic.style.transform = "scaleX(-1)";`

If you can look at Dee's homework. It is a simple version of the catwalk but its easy to understand how to make it walk back and forth